docs/tools/vdb_table/data/cratedb.json

```diff
@@ -29,8 +29,8 @@
   "vector_launch_year": 2023,
   "metadata_filter": {
     "support": "full",
-    "source_url": "https://cratedb.com/docs/crate/reference/en/5.5/general/builtins/scalar-functions.html",
-    "comment": "combine knn_match with other criteria"
+    "source_url": "https://cratedb.com/docs/crate/reference/en/latest/general/builtins/scalar-functions.html",
```

Comment:
I think this would be the appropriate url for this feature: https://cratedb.com/docs/crate/reference/en/latest/general/builtins/comparison-operators.html

"metadata_filter" refers to being able to filter rows based on boolean criteria

Reply:
I think this needs to be edited

Reply:
ok